### PR TITLE
add project description on PyPI

### PR DESCRIPTION
**Pull request by lostanlen**
fixes ~#36~ #151 (sorry, i initially wrote #36 because i was working on birdvoxdetect at the same time!)

update description, long_description and classifiers

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
 
+with open('README.md', 'r') as fdesc:
+    long_description = fdesc.read()
+
 if __name__ == '__main__':
     setup(
         name='mirdata',
         version='0.1.1',
-        description='Common loaders for MIR datasets.',
+        description='Data management utilities for reproducible research in music information retrieval (MIR).',
```
Comment (magdalenafuentes) [inline setup.py]: I got used to the "common loaders for reproducible...", but this works for me too

Comment (lostanlen) [reply setup.py]: It's not clear what "common loaders" mean. "Common"= "widespread" or "common"="compatible"?

Comment (lostanlen) [reply setup.py]: @magdalenafuentes note that @rabitt approved this change, so if you're OK with this `description` string, we can resolve the conversation and merge this PR.
This is not an irreversible design decision, so it's fine to GTD and fine-tune later.

**Comment by lostanlen** [comment]
> If I understand well this reads from the README right? What happens if we add the .gif in the README? will this one render it too?

Yes @magdalenafuentes, this hooks into the README and uses it as a README on PyPI.
I think it's OK. PyPI can render GIFs.

One example that comes to mind is tqdm, which has a GIF logo:
https://github.com/tqdm/tqdm

https://pypi.org/project/tqdm/

Does this answer your Q? shall we merge this?

**Comment by magdalenafuentes** [comment]
yes let's merge!

**Comment by lostanlen** [comment]
I set version to `0.2.dev0` in accordance with PEP 440
https://www.python.org/dev/peps/pep-0440/

scikit-learn does the same

**Comment by lostanlen** [comment]
@rabitt this PR is next in line. it explicitly removes py27 support

**Pull request merged**